rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -243,4 +241,49 @@ private static boolean isFromFeePerKbChangeAuthorizedSender(org.ethereum.core.Tr
         AddressBasedAuthorizer authorizer = bridgeConfiguration.getFeePerKbChangeAuthorizer();
         return authorizer.isAuthorized(rskTx);
     }
+
+    public static boolean validateHeightAndConfirmations(int height, int btcBestChainHeight, int acceptableConfirmationsAmount, Sha256Hash btcTxHash) throws Exception {
+        // Check there are at least N blocks on top of the supplied height
+        if (height < 0) {
+            throw new Exception();
```

Comment:
Maybe it would be better to throw a custom exception

Reply:
It could be a VerificationException maybe

Reply:
Yes, we should throw meaningful exceptions. Typed and/or with a message to help the log reader grasp what happened.

Although in this case you could just log and return false. No need to throw an exception I think.

Reply:
Throws exception with message now, as it was throwing exception originally. In case we prefer to log and return false, we should remove the try/catch block on the call.